File: 2st/Yeonwoo/더 크게 합치기.py
def solution(a, b):
    answer = 0
    str_a_b = str(a)+str(b)
    str_b_a = str(b)+str(a)
    if int(str_a_b) > int(str_b_a):
        answer = int(str_a_b)
    elif int(str_a_b) < int(str_b_a):
        answer = int(str_b_a)
    elif int(str_a_b) == int(str_b_a):
        answer = int(str_a_b)
        
    return answer

File: 4st/Eunseong/더 크게 합치기.java
class Solution {
    public int solution(int a, int b) {
        int answer = 0;
        String sum1 = a + Integer.toString(b);
        String sum2 = b + Integer.toString(a);
        
        if (Integer.valueOf(sum1) >= Integer.valueOf(sum2)) {
            return Integer.valueOf(sum1);
        } else {
            return Integer.valueOf(sum2);
        }
    }
}